### PR TITLE
add documentation for `toTitleCase()` second argument (delimiters)

### LINKED ISSUE #6685: toTitleCase() second argument (delimiters) undocumented
As reported by @ostephens here: https://github.com/OpenRefine/OpenRefine/issues/5894#issuecomment-1576430234

The `delimiters` argument to `toTitleCase()` is undocumented in the bundled help.

### PR DESCRIPTION
**Pull request by akashinde**
Fixes #6685

Changes proposed in this pull request:
- When transforming values of column cells, in **Help** tab -> Documentation of **toTitlecase()** function is updated with latest description from user manual.

Post changes steps taken:
- [x] run lint
- [x] run cypress tests (optional as changes were not related to frontend) (PASSED)
- [x] run java tests (PASSED)

Final result:
image

### CODE REVIEW

File: main/resources/com/google/refine/grel/FunctionDescription.properties [hunk 1]
```diff
@@ -93,7 +93,7 @@ str_split_by_lengths=Returns the array of strings obtained by splitting s into s
 str_split=Returns the array of strings obtained by splitting s by sep. The separator can be either a string or a regex pattern. If preserveTokens is true, empty segments are preserved.
 str_starts_with=Returns a boolean indicating whether s starts with sub. For example, "food".startsWith("foo") returns true, whereas "food".startsWith("bar") returns false.
 str_to_lowercase=Returns string s converted to all lowercase characters.
-str_to_title_case=Returns string s converted into titlecase: a capital letter starting each word, and the rest of the letters lowercase. For example, ''Once upon a midnight DREARY''.toTitlecase() returns the string ''Once Upon A Midnight Dreary''.
+str_to_title_case=Returns string s converted into titlecase: a capital letter starting each word, and the rest of the letters lowercase. For example, ''Once upon a midnight DREARY''.toTitlecase() returns the string ''Once Upon A Midnight Dreary''. Optionally, a string of delimiter characters used to separate words. The first character and the first non-delimiter character after a delimiter will be captialized. For example, "jean-michel jarre".toTitlecase(" -") returns "Jean Michel Jarre".
```
Comment (tfmorris) [inline main/resources/com/google/refine/grel/FunctionDescription.properties:96]:
```suggestion
str_to_title_case=Returns string s converted into titlecase: a capital letter starting each word, and the rest of the letters lowercase. For example, ''Once upon a midnight DREARY''.toTitlecase() returns the string ''Once Upon A Midnight Dreary''. Optionally, a string of delimiter characters used to separate words. The first character and the first non-delimiter character after a delimiter will be captialized. For example, "jean-michel jarre".toTitlecase(" -") returns "Jean-Michel Jarre".
```

**Review by tfmorris — CHANGES_REQUESTED** [review]
Thanks for the contribution! And apologies for the delay in reviewing. 

It looks pretty good, but I get a different result for your example when I test on my system. Other than that, it looks perfect.

File: main/resources/com/google/refine/grel/FunctionDescription.properties (hunk 1 above)
Comment (akashinde) [reply main/resources/com/google/refine/grel/FunctionDescription.properties:96]: Thanks for reviewing the changes. Also, I copied the description from the User manual and I think we might have to change that as well.

https://openrefine.org/docs/manual/grelfunctions#totitlecases

File: main/src/com/google/refine/expr/functions/strings/ToTitlecase.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public String getDescription() {
 
     @Override
     public String getParams() {
-        return "string s";
+        return "string s, string delimiter (optional)";
```
Comment (tfmorris) [inline main/src/com/google/refine/expr/functions/strings/ToTitlecase.java:74]:
```suggestion
        return "string s, string delimiters (optional)";
```

**Review by tfmorris — APPROVED** [review]
Looks great. Thank you very much!

Yes, I realized afterward that the same issue existed in doc site (because I never submitted my review with the correction. Oops!)

**Pull request merged**